packages/loader/src/gltf/extensions/MeshoptDecoder.ts

```diff
@@ -10,7 +10,15 @@ interface WorkerItem {
   requests: Record<string, any>;
 }
 
-const MeshoptDecoder = (function () {
+let ready: Promise<{
```

Comment:
Why not named:`instance`

Reply:
The core decoder code is copy from https://github.com/zeux/meshoptimizer/blob/master/js/meshopt_decoder.js. It's unnecessary to change the name style of the original code.

Reply:
OK